home/web/client/reducers/pendingTransaction.js

```diff
@@ -0,0 +1,48 @@
+import { handleActions } from 'redux-actions';
+
+const initialState = {
+  isSending: false,
+  open: false,
+  error: '',
+  transaction: {},
+  callback: null
+};
+
+export default handleActions({
+
+  'update pendingTransaction' (state, action) {
```

Comment:
Maybe change to `update unsignedTransaction`
to be consisted with rest of methods (and more accurate?)